apps/guardian-ui/src/components/BalanceTable.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Table,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  Icon,
+  Box,
+  Flex,
+} from '@chakra-ui/react';
+import { ReactComponent as CheckIcon } from '../assets/svgs/check-circle.svg';
+import { ReactComponent as CloseIcon } from '../assets/svgs/x-circle.svg';
+import { AuditSummary, ModuleKind, MSats } from '@fedimint/types';
+import { formatMsatsToBtc } from '@fedimint/utils';
+
+interface BalanceTableProps {
+  auditSummary: AuditSummary;
+  unit: 'msats' | 'sats' | 'btc';
+}
+
+export const BalanceTable: React.FC<BalanceTableProps> = ({
+  auditSummary,
+  unit,
+}) => {
+  const moduleSummaries = Object.entries(auditSummary.module_summaries).filter(
+    ([, module]) => module.kind !== ModuleKind.Unknown
+  );
+
+  const totalAssets = moduleSummaries.reduce((sum, [, module]) => {
+    return sum + (module.net_assets > 0 ? module.net_assets : 0);
+  }, 0) as MSats;
+  const totalLiabilities = moduleSummaries.reduce((sum, [, module]) => {
+    return sum + (module.net_assets < 0 ? module.net_assets : 0);
+  }, 0) as MSats;
+  const isBalanced = totalAssets + totalLiabilities === 0;
```

Comment:
This will be false for any federation charging fees. I think as long as assets > liabilities we should say it's balanced.

Reply:
We should probably have a way of specifically separating out the fees either into an "equity" column or a way to get the specific amounts collected in fees from the different modules so that we can still display that the outstanding liabilities match the outstanding reserves exactly.

Reply:
Modules don't collect fees themselves and we don't record fees anywhere. All that happens is that the federation slowly becomes overfunded. We discussed different approaches, but this didn't seem like a hug problem so far. If you want you can just display any overfunding as a fees entry on the liabilities side.